readme.md

```diff
@@ -119,15 +143,25 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+Save bang will throw an error, whereas save without a bang will fail silently.
```

Comment:
Technically, save-bang will throw a fatal exception, not an error. save-bangless still generates an error, kept in `@artist.errors`, but it's not fatal.